utils/node_local_test.py

```diff
@@ -3341,6 +3410,21 @@ def test_daos_fs_fix(self):
         if line[-1] != 'Setting entry type to S_IFDIR':
             raise NLTestFail('daos fs fix-entry /test_dir/1d2')
 
+        # remount dfuse
+        dfuse = DFuse(self.server,
+                      self.conf,
+                      pool=self.pool.id(),
+                      container=self.container,
+                      caching=False)
+        dfuse.log_mask = 'CRIT'
+        dfuse.start(v_hint='fs_fix_test')
```

Comment:
Can we not re-use the same dfuse object and call `dfuse.start(...)` ?

Reply:
yea i had that first but did not work. maybe i missed something.

Reply:
ah i remember now. the start assumes the path for the dfuse mountpoint is created (part of the class init i assume). the stop if removing that path. so if you call start after stop, it fails because the path was removed.
So the init is needed again to create a new path for the dfuse mountpoint. which is why in the other test, i had to reconstruct the file names with the new dfuse path base prefix.
probably something that can be improved by separating moving the rmdir into a delete and not close..

Reply:
Restarting a dfuse process from the same object isn't something I'd considered and no, I wouldn't expect this to work.  To me I associate a DFuse object with a **process** so in that context I wouldn't expect it to work but I can see your point.

Reply:
Gotcha - not a blocker for me. I'll defer to Ashley for utilities in this file :)